chpl/chpl-service/src/main/java/gov/healthit/chpl/dto/DecertifiedDeveloperDTO.java

```diff
@@ -46,15 +46,27 @@ public Long getNumMeaningfulUse() {
 	}
 
 	public void setNumMeaningfulUse(Long numMeaningfulUse) {
-		this.numMeaningfulUse = numMeaningfulUse;
+		if(this.numMeaningfulUse == null){
+			this.numMeaningfulUse = numMeaningfulUse;
+		}
+		else{
+			this.numMeaningfulUse += numMeaningfulUse;
+		}
+		
 	}
 	
 	public void addAcb(String acb){
 		this.oncacb.add(acb);
 	}
 	
 	public void addNumMeaningfulUse(Long numMeaningfulUse){
-		this.numMeaningfulUse+= numMeaningfulUse;
+		if(this.numMeaningfulUse != null){
```

Comment:
I'm confused why the setter and this add method are doing the same thing (although the logic is reversed). Maybe the setter didn't need to be changed?